### PR TITLE
Sync published with master

### PR DESCRIPTION
**Pull request by usha-mandya**

### CODE REVIEW

**Comment by usha-mandya** [comment]
@ollypom - Could you PTAL?

File: app/working-with-app.md [hunk 1]
```diff
@@ -372,57 +372,60 @@ port `8080` is allowed on the connection form your browser to your Docker host.
...
-As mentioned in the introduction, `docker app` lets you manage entire applications the same way that you 
-currently manage container images. For example, you can push and pull entire applications from registries like 
-Docker Hub with `docker app push` and `docker app pull`. Other `docker app` commands, such 
-as `install`, `upgrade`, `inspect` and `render` can be performed directly on applications while they are 
-stored in a registry.
+As mentioned in the introduction, `docker app` lets you manage entire
+applications the same way that you currently manage container images. For
+example, you can push and pull entire applications from registries like Docker
+Hub with `docker app push` and `docker app pull`. Other `docker app` commands,
+such as `install`, `upgrade`, `inspect` and `render` can be performed directly
```
Comment (traci-morrison) [inline app/working-with-app.md]: `inspect`, and `render`

File: app/working-with-app.md [hunk 2]
```diff
@@ -372,57 +372,60 @@ port `8080` is allowed on the connection form your browser to your Docker host.
...
+on applications while they are stored in a registry.
+
+Push the application to Docker Hub. To complete this step, you need a valid
+Docker ID and you must be logged in to the registry to which you are pushing
+the app.
+
+By default all platform architectures are pushed to the registry. If you are
```
Comment (traci-morrison) [inline app/working-with-app.md]: "By default,..."

File: app/working-with-app.md [hunk 3]
```diff
@@ -372,57 +372,60 @@ port `8080` is allowed on the connection form your browser to your Docker host.
...
+pushing an official Docker image as part of your app, you may find your app
+bundle becomes large with all image architectures embedded. To just push the
+architecture required you can add the `--platform` flag.
```
Comment (traci-morrison) [inline app/working-with-app.md]: "To just push the architecture required, you can add the `--platform` flag."

File: app/working-with-app.md [hunk 4]
```diff
@@ -372,57 +372,60 @@ port `8080` is allowed on the connection form your browser to your Docker host.
...
+[pushing an app to Docker Hub](#push-the-app-to-docker-hub) except that you
+need your DTR user credentials and [your DTR repository
+information](/ee/dtr/user/manage-images/review-repository-info/). To use client
+certificates for DTR authentication, see [Enable Client Certificate
+Authentication](/ee/enable-client-certificate-authentication/).
+
+By default all platform architectures are pushed to DTR. If you are pushing an
+official Docker image as part of your app, you may find your app bundle
+becomes large with all image architectures embedded. To just push the
+architecture required you can add the `--platform` flag.
```
Comment (traci-morrison) [inline app/working-with-app.md]: I would apply the previous suggestions to this section.

File: app/working-with-app.md (hunk 1 above)
Comment (usha-mandya) [reply app/working-with-app.md]: Fixed in https://github.com/docker/docker.github.io/pull/9170

File: app/working-with-app.md (hunk 2 above)
Comment (usha-mandya) [reply app/working-with-app.md]: Fixed in #9170

File: app/working-with-app.md (hunk 3 above)
Comment (usha-mandya) [reply app/working-with-app.md]: Fixed in #9170

File: app/working-with-app.md (hunk 4 above)
Comment (usha-mandya) [reply app/working-with-app.md]: Fixed in #9170

**Pull request merged**